lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -357,12 +358,17 @@ class BaasClient {
           }
         ]
       }''',
-    );
-    await _put('groups/$_groupId/apps/$appId/sync/config', '{ "development_mode_enabled": true }');
-
-    //create email/password user for tests
-    final dynamic createUserResult = await _post('groups/$_groupId/apps/$appId/users', '{"email": "realm-test@realm.io", "password":"123456"}');
-    print("Create user result: $createUserResult");
+      );
+      await _put('groups/$_groupId/apps/$appId/sync/config', '{ "development_mode_enabled": true }');
+
+      //create email/password user for tests
+      final dynamic createUserResult = await _post('groups/$_groupId/apps/$appId/users', '{"email": "realm-test@realm.io", "password":"123456"}');
+      print("Create user result: $createUserResult");
+    } catch (error) {
+      print(error);
```

Comment:
Do we need to print the error here?

Reply:
Yes, I would like to see which of the actions of app services creation are passed successfully before the error.

Reply:
I agree we can leave it for the test.dart but here I think we should not `print`.
If you want to see which apps succeeded you can print them in the tests.dart